.github/workflows/release.yml

```diff
@@ -346,6 +346,14 @@ jobs:
         run: |
           docker run --rm test-${{ matrix.distro }}
 
+  build-plugin-compiler:
```

Comment:
Is it possible to remove this from release.yml altogether? Because keeping it here means that a explicit dependency between the templates and the plugin compiler will continue. All we need is that .github/workflows/plugin-compiler-build.yml is triggered when the release workflow runs.
From https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-branches-for-workflow-run-events there appears to be a way to do this.
> For example, a workflow with the following trigger will only run when the workflow named Build runs on a branch whose name starts with releases/:
```
on:
  workflow_run:
    workflows: ["Build"]
    types: [requested]
```